src/components/Table/spec.js.snap

```diff
@@ -52,12 +52,18 @@ exports[`Table component should match the stored snapshot 1`] = `
   display: table-cell;
   border-bottom: 1px solid #DDE1f0;
   text-align: left;
-  padding-left: 40px;
-  padding-top: 10px;
-  padding-bottom: 10px;
+  padding: 10px 20px;
```

Comment:
NIT: We can also make it `space[2] space[4]` since we are changing it anyway, but it's not a big deal.